chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,72 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate menulist with locales
+	 */
```

Comment:
I think we're talking past each other. The question was
>do we want to move the preferred/fallback locales to the top of the list for easy selection or no?

(Regardless of whether they are recognized or not)